.htaccess

```diff
@@ -90,3 +94,5 @@
 AddType video/ogg .ogv
 AddType video/mp4 .mp4
 AddType video/webm .webm
+
+## END Known Webserver Config (don't remove)
```

Comment:
Let's add another newline here.

Reply:
Probably best duplicate this in htaccess.dist and htaccess-2.4.dist